databricks/koala/structures.py

```diff
@@ -239,6 +239,18 @@ def _metadata(self):
             self._pandas_metadata = ref._metadata.copy(column_fields=[self.name])
         return self._pandas_metadata
 
+    @_metadata.setter
+    def _metadata(self, metadata):
+        self._pandas_metadata = metadata
+
+    @property
+    def index(self):
+        """The index (axis labels) Column of the Series.
+
+        Currently supported only when the DataFrame has a single index.
+        """
+        return self._pandas_anchor.index
```

Comment:
shouldn't there be a check for a single index? The message should also mention what happens when there is more than one index.